private-templates-service/adaptivecards-templating-service/src/util/inmemorydbutils/inmemorydbutils.ts

```diff
@@ -0,0 +1,29 @@
+import { SortOrder } from "../../models/models";
+
+export function clone<T>(obj: T): T {
+  let cloned: T = JSON.parse(JSON.stringify(obj));
+  return cloned;
+}
+
+export function ifContainsList<T>(toVerify: T[], list: T[]): boolean {
+  if (!list.length) {
```

Comment:
`if (list.length !== toVerify.length) { return false; }` ?

Reply:
they don't need to mach exactly, it's enough for **toVerify** list to contain all the tags provided in the list